src/dynamic_batch_scheduler.cc

```diff
@@ -315,7 +330,7 @@ DynamicBatchScheduler::BatcherThread(const int nice)
     NVTX_RANGE(nvtx_, "DynamicBatcher " + model_name_);
 
     std::shared_ptr<std::vector<std::deque<std::unique_ptr<InferenceRequest>>>>
-        rejected_requests;
+        rejected_requests, cancelled_requests;
```

Comment:
do these need to be shared pointers? I only see them used locally and not returned - maybe missing something though

Reply:
I agree they (`rejected_requests`, `cancelled_requests`) do not need to be shared pointers, but just in case, @GuanLuo @tanmayv25 do you recall why `rejected_requests` is a shared pointer in the first place?

Reply:
The L0_batcher test passed after removing the shared pointer wrapping.

Reply:
I am not sure... but I think we should be fine.